### PR TITLE
Enable vertical resizing for multiline InputBase

### PR DESCRIPTION
**Pull request by kerstin97**
## Description

Enable vertical resizing for `TextAreaField`

## Example

-   [x] I have verified if my change requires an example

## Screenshots/screencasts

| Before   | After   |
| -------- | ------- |
| image     |   <video src="https://github.com/user-attachments/assets/8e16faa0-cc63-424c-a1da-605270adeabd" /> |

## Changeset

-   [x] I have verified if my change requires a changeset

## Related tasks and documents

https://vivid-planet.atlassian.net/browse/COM-1268

## Open TODOs/questions

- [x] Wait for this fix before merging to avoid broken styling: https://vivid-planet.atlassian.net/browse/COM-1265

### CODE REVIEW

File: .changeset/fluffy-jars-search.md
Comment (jamesricky) [inline .changeset/fluffy-jars-search.md:2]: I'd say this is a new feature, not a bug 🙂 

```suggestion
"@comet/admin-theme": minor
```

File: .changeset/fluffy-jars-search.md
Comment (jamesricky) [inline .changeset/fluffy-jars-search.md:5]: Technically it affects any multiline input

```suggestion
Enable vertical resizing for multiline inputs
```

```suggestion
Enable vertical resizing for `TextAreaField` and other multiline inputs
```

**Review by jamesricky — COMMENTED** [review]
Please wait for this fix to be merged, before merging: https://vivid-planet.atlassian.net/browse/COM-1265

Otherwise it looks broken, due to the spacing on the right:
image

**Review by jamesricky — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Review by jamesricky — COMMENTED** [review]
Please check if this still works correctly and the styling is fixed, as https://github.com/vivid-planet/comet/pull/2857 has been merged. Thanks 🙂

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**
